src/main/features/response/form/models/statement-of-means/employers.ts

```diff
@@ -28,33 +28,11 @@ export class Employers implements Serializable<Employers> {
     return new Array(rows).fill(EmployerRow.empty())
   }
 
-  deserialize (input?: any): Employers {
-    if (input) {
-      this.rows = this.deserializeRows(input.rows)
-    }
-
-    return this
-  }
-
-  appendRow () {
-    if (this.canAddMoreRows()) {
-      this.rows.push(EmployerRow.empty())
-    }
-  }
-
-  removeExcessRows () {
-    this.rows = this.rows.filter(item => !!item.jobTitle && !!item.employerName)
-
-    if (this.rows.length === 0) {
-      this.appendRow()
-    }
+  getMaxNumberOfRows (): number {
```

Comment:
That could be just property getter that can be overridden instead of forcing method syntax.